src/screens/attendance/DailyAttendance.js

```diff
@@ -0,0 +1,130 @@
+import React, { useEffect, useState } from 'react';
+
+import {
+    View,
+    Text,
+    StyleSheet,
+    TouchableOpacity,
+    Alert
+} from 'react-native';
+import { useAppSelector } from '../../../store/hook';
+import firestore from '@react-native-firebase/firestore';
+import DropDownPicker from 'react-native-dropdown-picker';
+import styles from './DailyAttendance.styles';
+
+const DailyAttendance = ({ navigation }) => {
+    const user = useAppSelector(state => state.profile.data);
+    const [subjects, setSubjects] = useState([]);
+    const [subject, setSubject] = useState(null)
```

Comment:
rename subject to selectedSubject
rename setSubject to setSelectedSubject

It should be like :
const [selectedSubject, setSelectedSubject] = useState(null)